packages/sit-onyx/src/components/examples/GridPlayground/GridPlayground.vue

```diff
@@ -78,99 +88,117 @@ const updateElement = (index: number, newElement: GridElementConfig) => {
+        For the best experience, please press the "Fullscreen" button in the upper right corner.
       </p>
 
       <p>
-        For further details on the grid, please refer to our
-        <OnyxLink href="https://onyx.schwarz/development/grid.html" target="_blank"
-          >grid docs</OnyxLink
-        >
+        This is the playground for grid and breakpoint usage. Feel free to create placeholder
+        objects play around with them. To see the responsiveness of the onyx grid in action, just
+        use the window resizer to adjust the width of your browser. You can globally adjust the grid
+        the way that fit your needs. Details about the technical implementation can be found
+        <OnyxLink href="https://onyx.schwarz/development/grid.html" target="_blank">here</OnyxLink>
       </p>
 
-      <p>
-        <strong
-          >For the best experience, please press the "Go full screen" button in the upper right
-          corner.</strong
-        >
-      </p>
+      <OnyxHeadline is="h2" class="playground__headline">Grid customization</OnyxHeadline>
 
       <div class="playground__options">
-        <OnyxRadioGroup
-          v-model="gridSettings.maxWidth"
-          label="Max content width"
-          :options="maxWidthOptions"
-          direction="horizontal"
-        />
-
-        <OnyxRadioGroup
+        <OnyxSelect
           v-model="gridSettings.alignment"
-          label="Alignment"
-          label-tooltip="In case of a grid with a max content width of 'none', this determines the alignment of the grid elements as a whole."
+          label="Grid alignment"
+          list-label="List of alignment options"
+          label-tooltip="You can adjust the overall alignment of the grid here."
           :options="alignmentOptions"
-          direction="horizontal"
-          :disabled="gridSettings.maxWidth === 'none'"
         />
 
-        <OnyxRadioGroup
+        <OnyxSelect
+          v-model="gridSettings.maxWidth"
+          label="Max overall width"
+          list-label="List of max width options"
+          label-tooltip="With this setting, you can adjust the maximum width of the container that includes the content. This is only relevant for large breakpoints."
+          :options="maxWidthOptions"
+          :readonly="!largeBreakpoint"
+        />
+
+        <OnyxSelect
           v-model="gridSettings.maxColumns"
-          label="Max columns"
-          label-tooltip="Defines the maximum number of columns the grid can have on the 'LG' and 'XL' breakpoints. Per default the grid is restricted to 12 columns. "
+          label="Column quantity"
+          list-label="List of max columns options"
+          label-tooltip="With large breakpoints you can optionally extend the default 12 column grid to 16 or even 20 columns."
           :options="maxColumnsOptions"
-          direction="horizontal"
+          :readonly="!largeBreakpoint"
```

Comment:
There should only be options for 12 (default)/16/20 columns. Also shouldn't be set to readonly, we just should use a more specific label.
```suggestion
          v-model="gridSettings.maxColumns"
          label="XL breakpoint column count"
          list-label="List of columns options"
          label-tooltip="With large breakpoints you can optionally extend the default 12 column grid to 16 or even 20 columns."
          :options="maxColumnsOptions"
```